app/views/layouts/react_page.haml

```diff
@@ -1,4 +1,108 @@
-= render :partial => 'layouts/head'
+!!! 5
+
+%head
+  %meta{charset: "utf-8"}
+
+  -# Kissmetrics is deprecated, use Google Tag Manager instead
+  = render :partial => "layouts/kissmetrics"
```

Comment:
Who knows about this? Is this needed? The comment says it is deprecated and GTM should be used instead and Tag Manager is used on the next line.